portfolio/src/main/webapp/script.js

```diff
@@ -118,35 +119,45 @@ function showModal() {
     let commentForm = document.createElement("form");
```

Comment:
I'm not sure why neither @jalexanderqed nor I commented on this...

This type of form should normally be in your HTML, with styling `display:none`. When commentbutton is clicked, as you're doing here, you can remove the `display:none` styling and it will become visible.

Constructing large amounts of html out of whole cloth in javascript is perilous and harder to maintain unless you're using [a UI library that specifically works like that](https://reactjs.org/)